frontend/dispatcher.lua

```diff
@@ -286,50 +319,89 @@ local dispatcher_menu_order = {
     "embedded_fonts",
     "smooth_scaling",
     "nightmode_images",
+
+    "kopt_trim_page",
+    "kopt_page_margin",
+
+    "kopt_zoom_overlap_h",
+    "kopt_zoom_overlap_v",
+    "kopt_zoom_mode_type",
+    --"kopt_zoom_range_number", -- can'y figure out how this name text func works
+    "kopt_zoom_factor",
+    "kopt_zoom_mode_genus",
+    "kopt_zoom_direction",
+
+    "kopt_page_scroll",
+    "kopt_page_gap_height",
+    "kopt_full_screen",
+    "kopt_line_spacing",
+    "kopt_justification",
+
+    "kopt_font_size",
+    "kopt_font_fine_tune",
+    "kopt_word_spacing",
+    --"kopt_text_wrap", -- has multiple events
+
+    "kopt_contrast",
+    "kopt_page_opt",
+    "kopt_hw_dithering",
+    "kopt_quality",
+
+    "kopt_doc_language",
+    "kopt_forced_ocr",
+    "kopt_writing_direction",
+    "kopt_defect_size",
+    "kopt_auto_straighten",
+    "kopt_detect_indent",
+    "kopt_max_columns",
 }
 
 --[[--
     add settings from CreOptions / KoptOptions
 --]]--
 function Dispatcher:init()
     if Dispatcher.initialized then return end
-    local parseoptions = function(base, i)
+    local parseoptions = function(base, i, prefix)
         for y=1,#base[i].options do
```

Comment:
(Space after comma, also below - but you may not want to have this in your diff.)